### PR TITLE
chore: persist menu in DOM

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-7878

### CODE REVIEW

File: src/components/Menu/MenuClickAwayListener.tsx
Comment (dennyscode) [inline src/components/Menu/MenuClickAwayListener.tsx:1]: This is unfortunate, but I wasted too much time with the MUI clickawaylistener and decided to drop it in favor of a component which handles just that.

Comment (tcheee) [reply src/components/Menu/MenuClickAwayListener.tsx:1]: Would prefer to remove the div to be able to keep the click on the widget and not the double click

File: src/components/Navbar/NavbarButtons/NavbarButtons.tsx [hunk 1]
```diff
@@ -38,36 +39,48 @@ export const NavbarButtons = ({ redirectToLearn }: NavbarButtonsProps) => {
...
+        >
+          <MenuIcon
+            sx={{
+              fontSize: '32px',
+              color: 'inherit',
+            }}
+          />
+        </MenuToggle>
+      </NavbarButtonsContainer>
       <MainMenu anchorEl={mainMenuAnchor.current} />
```
Comment (tcheee) [inline src/components/Navbar/NavbarButtons/NavbarButtons.tsx:83]: why changing the JSX org on this?

File: src/components/Navbar/WalletButton.tsx [hunk 2]
```diff
@@ -82,7 +82,10 @@ export const WallettButtons = () => {
         <ConnectButton
           // Used in the widget
           id="connect-wallet-button"
-          onClick={handleWalletSelectClick}
+          onClick={(event) => {
+            event.stopPropagation();
```
Comment (tcheee) [inline src/components/Navbar/WalletButton.tsx:86]: Why adding this? Is it related to the clickAwayListener?

**Comment by tcheee** [comment]
Small bug found on the feature branch when clicking on the menu, the page is scrolling to the bottom.

**Review by tcheee — APPROVED** [review]

**Pull request merged**